site/src/pages/components/accordion.explainer.mdx

```diff
@@ -0,0 +1,157 @@
+---
+menu: Proposals
+name: Exclusive Accordion (Explainer)
+path: /components/accordion.explainer
+pathToResearch: /components/accordion.research
+layout: ../../layouts/ComponentLayout.astro
+---
+
+- [@dbaron](https://github.com/dbaron)
+- Last updated: April 21, 2023
+
+{/* START doctoc generated TOC please keep comment here to allow auto update */}
+{/* DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE */}
+
+## Table of Contents
+
+- [Background](#background)
+- [Developer Demand for Exclusive Accordion](#developer-demand-for-exclusive-accordion)
+- [Proposal](#proposal)
+  - [Open design questions](#open-design-questions)
+    - [Name](#name)
+    - [Interaction with `open` attribute](#interaction-with-open-attribute)
+    - [Use case for non-exclusive accordion semantic?](#use-case-for-non-exclusive-accordion-semantic)
+  - [Possibly-resolved design questions](#possibly-resolved-design-questions)
+    - [Scope](#scope)
+- [Alternatives considered](#alternatives-considered)
+  - [CSS Toggles](#css-toggles)
+
+# Background
+
+A **disclosure** widget
+([research](/components/disclosure.research))
+is a UI that shows a summary or heading along
+with an indicator (sometimes a triangle or a plus sign) that allows the
+widget to be expanded to show additional details.
+An **accordion**
+([research](/components/accordion.research))
+is a sequence of related disclosure widgets.
+Some accordions are **exclusive accordions**,
+which means that at most one of the disclosures in that accordion
+can be open at the same time.
+
+The existing HTML
+[`details` element](https://html.spec.whatwg.org/multipage/C#the-details-element)
+represents a disclosure widget.
+A sequence of `details` elements can be used for a non-exclusive accordion.
+However, there is not a current web platform feature
+for exclusive accordions.
+
+It is worth noting that the HTML `details` element
+is not currently as styleable as it needs to be to meet many use cases.
+Improving styleability of `details` will be addressed in a separate proposal,
```

Comment:
It'd be helpful to have a "For example, you can't style ..." sentence right here.

Reply:
Added (with somewhat different wording).